### ISSUE TITLE
Cant set the right filter

### ISSUE THREAD

**Issue by Kongux** [oRastor/fut21-web-app#69]
I don't know, if it is a general problem of the web app... but when I select gold non rare for example I find everything... bronze cards, silver and gold. Does someone has the same Problem?

**Comment by ab-user1** [comment]
yes, seems like this owner doesnt care about this autobuyer anymore..

its a commong issue among all bots and only this guy doesnt update his bot

**Comment by oRastor** [comment]
![screenshot-www ea com-2021 05 06-17_12_39](https://user-images.githubusercontent.com/1734394/117313187-56378400-ae8e-11eb-9a99-6a19f37c22ed.png)
![screenshot-www ea com-2021 05 06-17_13_00](https://user-images.githubusercontent.com/1734394/117313190-5768b100-ae8e-11eb-94c3-a5ff68193c7e.png)

It's a common problem of web app... Do you want to generate a request from a web app that cannot generate a basic web app? It can be simply detected by EA servers
